charts/alertmanager/Chart.yaml

```
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 1.17.2
+version: 1.17.3
```

new feature shuld raise minor version
```suggestion
version: 1.18.0
```